Feature/out folder name (#3)
cancerit/workflow-seq-import: README.md

#### Inputs

> * `fastq_in` - An gzipped interleaved FastQ file.

Reviewer: ```suggestion
* `fastq_in` - A gzipped interleaved FastQ file.
```

* `chksum_post_server_response` - POST server response in a text file. Optional, if no `post_address` is given.
* `interleave_report_json`- A json report evaluating the analysis of the `fastq_in`.
* `interleave_ifastq_out` - Interleaved gzipped FastQ output file.

> * `rg_file_names` - A file which has the file name of the gzipped FastQ output file of the workflow. **NOTE:** The file will be useless if the workflow is ran by Dockstore, as the output file will be renamed by Dockstore.

Reviewer: ```suggestion
* `rg_file_names` - A file which has the file name of the gzipped FastQ output file of the workflow. **NOTE:** The file will be useless if the workflow is run by Dockstore, as the output file will be renamed by Dockstore.
```

### chksum_seqval_wf_paired_fq.cwl

#### Inputs

* `fastq_in` - A **list** of two paired gzipped FastQ files.

> * `post_address` - A **list** of two URLs to send checksum results of coresponding FasetQ to via POST. It's required and **NOT** optional, but two empty strings can be used if no POST is required.

Reviewer: ```suggestion
* `post_address` - A **list** of two URLs to send checksum results of corresponding FastQ to via POST. It's required and **NOT** optional, but two empty strings can be used if no POST is required.
```

* `chksum_post_server_response` - A list of two text files. Each has POST server response of one of the `fastq_in`. Optional if empty `post_address` was used.
* `interleave_report_json`- A json report evaluating the analysis of the `fastq_in`.
* `interleave_ifastq_out` - Interleaved gzipped FastQ output file.

> * `rg_file_names` - A file which has the file name of the gzipped FastQ output file of the workflow. **NOTE:** The file will be useless if the workflow is ran by Dockstore, as the output file will be renamed by Dockstore.

Reviewer: ```suggestion
* `rg_file_names` - A file which has the file name of the gzipped FastQ output file of the workflow. **NOTE:** The file will be useless if the workflow is run by Dockstore, as the output file will be renamed by Dockstore.
```
